### ISSUE TITLE
[BUG] [MODS] [1.15.2] Minecraft crashes upon entering Video Settings and another issue

### ISSUE THREAD

**Issue by SkywayzZarthic** [sp614x/optifine#3871]
## Description of Issue
**uses Forge 31.1.74 and also a whole bunch of mods, attached (unzipped) as well as preview_OptiFine_1.15.2_HD_U_G1_pre15.jar.**

1) When clicking to open video settings, the game crashes, shortly before or shortly after playing the sound that corresponds to clicking the button.
2) another issue, probably with forge, is items appearing dark in the GUI; however, this is not what this bug report is about, and there are other reports on it, so it will not be mentioned any further.

## Steps to Reproduce
1) launch Forge 31.1.74 with the attached mods.
2) go to options -> video settings
3) game crashes

## OptiFine Version
1.15.2 pre-15

## Installation Method
Dropped .jar into mods folder

## Fabric/Forge Version
Forge 31.1.74 (no OptiForge mod)

## Other Installed Mods
links are at the bottom- two of them, because otherwise it exceeded the limit

## Log Files/Crash Reports
Any relevant log files. Drag and drop text files here to upload to GitHub directly, OR upload the complete contents to Pastebin. https://pastebin.com/

You can get logs from `.minecraft/logs`, and crash reports from `.minecraft/crash-reports`. 
Locating your `.minecraft` folder will depend entirely on your OS. See this page for details: https://minecraft.gamepedia.com/.minecraft
**should be attached below as a link near the mods' links

## F3 Debug Screenshot
**this cannot be done as the issue occurs while paused - the F3 screen is not visible**

## Prior Testing
If you've installed using Forge/Fabric:
- Does this happen without other mods installed? 
   - Only OptiFine and Forge - 
No
- Does this happen without any mods at all?
   - Only Forge by itself.
No
- Does this happen in OptiFine standalone?
No
- Does this happen in Vanilla without OptiFine?
No
- Have you tried using a binary search to find which mods cause this issue?
   - To find conflicting mods, split your mods into 2 groups (not including OptiFine/OptiFabric, obviously). Remove one group, and test in-game. Keep the group that has the problem, and repeat until no more mods can be removed without the issue disappearing.
No I have not, because with all my mods and my laptop that could take hours.
   - Does the issue still happen with only the remaining mods installed?
      - Forge/Fabric and remaining mods, without OptiFine
N/A

## Additional Information
Here goes any final remarks and everything else that might not fit in the previous sections.
Right below are the necessary links- these are the mods and the log. No error shows up in the main menu on launch. I really hope that this issue is able to be fixed and isn't a bug with the Minecraft code itself.

[mods0.zip](https://github.com/sp614x/optifine/files/4583448/mods0.zip)
[journeymap-1.15.2-5.7.0b2.zip](https://github.com/sp614x/optifine/files/4583451/journeymap-1.15.2-5.7.0b2.zip)
[latest.log](https://github.com/sp614x/optifine/files/4583472/latest.log)

**Comment by naeqirelle2020** [comment]
@SkywayzZarthic does this occur in the latest preview?

**Comment by naeqirelle2020** [comment]
Tested and cannot reproduce this issue with forge + optifine most likely a mod conflict

**Comment by SkywayzZarthic** [comment]
Yes. I tested it out as well, and the mod conflict is TipTheScales. It
shouldn't have caused a crash, and the mod description says that it won't
work with optifine, not that it will crash, but it did, and it's all fine.
